WalletWasabi/BranchNBound/Selector.cs

```diff
@@ -0,0 +1,28 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.BranchNBound
+{
+	public class Selector
+	{
+		public Money CalcEffectiveValue(List<Money> list)
+		{
+			Money sum = Money.Satoshis(0);
+			if (list is not { })
+			{
+				return sum;
+			}
```

Comment:
This is really not that great because you have specified `List<Money> list` and not `List<Money>? list`. So this is really confusing.